src/NuGet.Indexing/ResponseFormatter.cs

```diff
@@ -346,7 +346,19 @@ public static void WriteStatsResult(JsonWriter jsonWriter, NuGetIndexSearcher se
             jsonWriter.WriteStartObject();
             WriteProperty(jsonWriter, "numDocs", searcher.IndexReader.NumDocs());
             WriteProperty(jsonWriter, "indexName", searcher.Manager.IndexName);
+            WriteProperty(jsonWriter, "machineName", searcher.Manager.MachineName);
             WriteProperty(jsonWriter, "lastReopen", searcher.LastReopen);
+            WriteProperty(jsonWriter, "lastIndexReloadTime", searcher.Manager.LastTimeIndexReloaded);
+            WriteProperty(jsonWriter, "lastIndexReloadDurationInMilliseconds", searcher.Manager.LastIndexReloadDurationInMilliseconds);
+            WriteProperty(jsonWriter, "lastAuxiliaryDataLoadTime", searcher.Manager.LastTimeAuxiliaryDataRefreshed);
```

Comment:
Earlier I was passing the values through a wrapper class which had the naming consistency, but I realized that passing through was not necessary. Aligning these properties now.